client/src/components/byline-slot/index.jsx

```diff
@@ -33,32 +35,42 @@ const BylineSlot = () => {
   const saveByline = createSaveByline(dispatch);
 
   useEffect(() => {
-    saveByline(profiles);
+    if (null !== profiles) {
+      saveByline(profiles);
+    }
   }, [profiles]);
 
   return (
     <div className="components-base-control">
-      <BylineAutocomplete
-        profiles={profiles}
-        onUpdate={addProfile}
-        profilesApiUrl={profilesApiUrl}
-        addAuthorPlaceholder={addAuthorPlaceholder}
-        addAuthorLabel={addAuthorLabel}
-      />
-      <BylineFreeform
-        onUpdate={addProfile}
-        addFreeformLabel={addFreeformLabel}
-        addFreeformPlaceholder={addFreeformPlaceholder}
-        addFreeformButtonLabel={addFreeformButtonLabel}
-      />
-      <BylineList
-        profiles={profiles}
-        onSortEnd={reorderProfile}
-        lockAxis="y"
-        helperClass="byline-list-item"
-        removeItem={removeProfile}
-        removeAuthorLabel={removeAuthorLabel}
-      />
+      {null === profiles ? (
+        <div style={{ textAlign: 'center' }}>
+          <Spinner />
+        </div>
+      ) : (
+        <Fragment>
+          <BylineAutocomplete
+            profiles={profiles}
+            onUpdate={addProfile}
+            profilesApiUrl={profilesApiUrl}
+            addAuthorPlaceholder={addAuthorPlaceholder}
+            addAuthorLabel={addAuthorLabel}
+          />
+          <BylineFreeform
+            onUpdate={addProfile}
+            addFreeformLabel={addFreeformLabel}
+            addFreeformPlaceholder={addFreeformPlaceholder}
+            addFreeformButtonLabel={addFreeformButtonLabel}
+          />
+          <BylineList
+            profiles={profiles}
+            onSortEnd={reorderProfile}
+            lockAxis="y"
+            helperClass="byline-list-item"
+            removeItem={removeProfile}
+            removeAuthorLabel={removeAuthorLabel}
+          />
+        </Fragment>
```

Comment:
Any reason not to use the shorthand syntax for fragments?

```suggestion
        <>
          <BylineAutocomplete
            profiles={profiles}
            onUpdate={addProfile}
            profilesApiUrl={profilesApiUrl}
            addAuthorPlaceholder={addAuthorPlaceholder}
            addAuthorLabel={addAuthorLabel}
          />
          <BylineFreeform
            onUpdate={addProfile}
            addFreeformLabel={addFreeformLabel}
            addFreeformPlaceholder={addFreeformPlaceholder}
            addFreeformButtonLabel={addFreeformButtonLabel}
          />
          <BylineList
            profiles={profiles}
            onSortEnd={reorderProfile}
            lockAxis="y"
            helperClass="byline-list-item"
            removeItem={removeProfile}
            removeAuthorLabel={removeAuthorLabel}
          />
        </>
```

Reply:
The build fails with `SyntaxError: Unexpected token` — possibly related to the Babel version as was the dependency conflict that led to `--legacy-peer-deps`.

Reply:
Bummer. We should fix that, but totally fine for it to be handled outside the scope of this issue.